examples/experimental/vz.yaml

```diff
@@ -1,3 +1,11 @@
+# ⚠️ ⚠️ ⚠️ `template://experimental/vz` will be removed in Lima v1.0,
```

Comment:
This looks misleading, During 1.0 its already removed then this file is not available right ? Maybe we need to bump the version ?

Reply:
Right. Will be removed in v1.0 RC.

Reply:
The CLI may print some hint when the removed template is specified

Reply:
Yes